src/include/platform/PlatformManager.h

```diff
@@ -332,6 +337,40 @@ inline CHIP_ERROR PlatformManager::InitChipStack()
     return err;
 }
 
+/**
```

Comment:
It would be really nice if this PR were scoped down to the changes really needed and the cleanups happened in a separate PR (that could then be reviewed to make sure it was moves only).  That would significantly aid review and understanding of what's changing when.

Reply:
I undid all the reordering improvements in this file.